extension-requests/local-utils.js

```diff
@@ -0,0 +1,101 @@
+async function getExtensionRequests(query = {}) {
+  const url = new URL(`${API_BASE_URL}/extension-requests`);
+
+  queryParams = ['assignee', 'status', 'taskId'];
+  queryParams.forEach(
+    (key) => query[key] && url.searchParams.set(key, query[key]),
+  );
+
+  const res = await fetch(url, {
+    credentials: 'include',
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json',
+    },
+  });
+  return await res.json();
+}
```

Comment:
can we wrap this in a `try catch` block

Reply:
the place where this function is called, the `try catch` is handled there